Aufgabe3/gta_v3/models/geotag.js

```diff
@@ -10,9 +10,49 @@
  * GeoTag objects should contain at least all fields of the tagging form.
  */
 class GeoTag {
+    #name = "";
+    #latitude = 0;
+    #longitude = 0;
+    #hashtag = "";
 
-    // TODO: ... your code here ...
-    
+    constructor(name, latitude, longitude, hashtag) {
```

Comment:
Muss ein hashtag übergeben werden? - Falls nein, weiterer constructor ohne hashtag und hashtag = null setzen?

Reply:
glaube es muss keiner übergeben werden, aber wir sollte trotzdem einen setzten. Also mein Vorschlag wäre ein zusätzlicher Konstruktor ohne Hashtag Argument und dann dort einfach den Namen zu nehmen und ein Hashtag anzuhängen.

Reply:
Dustin und ich haben uns dazu entschieden nur den einen Konstruktor stehen zu lassen, also wie ursprünglich. Da man in JavaScript nicht alle Argumente übergeben muss. Die nicht übergebenen Argumente werden einfach als undefined behandelt